src/garage/envs/multi_env_wrapper.py

```diff
@@ -204,13 +194,7 @@ def step(self, action):
 
         """
         es = self._env.step(action)
-
-        if self._mode == 'add-onehot':
-            obs = np.concatenate([es.observation, self._active_task_one_hot()])
-        elif self._mode == 'del-onehot':
```

Comment:
If 'del-onehot' mode is kept. We still need to remove these from obs?

Reply:
Is mode `del-onehot` ever used? IIUC only MultiEnvWrapper will populate the active task index in observation. After this PR, the index goes into `episode_info`. So I believe there is no need to remove task index from `obs` (aka mode `del-onehot`), and we can simply do `obs = es.observation` here